R/write_manifest.R

```diff
@@ -0,0 +1,46 @@
+write_manifest <- function(config, path, prior_input_paths = NULL) {
+  timestamp <-
+    as.character(
+      format(
+        x = Sys.time(),
+        format = "%Y-%m-%d %H:%M:%OS6",
+        tz = "UTC"
```

Comment:
if we enforce UTC time zone, we should probably document this somewhere

Reply:
would it make more sense to use the local timezone? I think that is achievable

Reply:
might be better, assuming the system this runs on actually has that set up appropriately. I remember an application with a supervisor last year, where the system was actually set to UTC although that was far off their actual timezone. somewhat confusing. I guess we could just use `Sys.time()` and also write `Sys.timezone()`?? as long as it is clearly spelled out, I am fine with either UTC or local

Reply:
If we leave out the `tz` specification entirely it defaults to `""` which is effectively the current "local" timezone. If a user's local timezone is set to UTC even if that's not geographically correct for them I think it is their problem.

Reply:
removed the explicit timezone specification

Reply:
okay, works for me